conf/minifi.properties

```diff
@@ -67,6 +67,7 @@ nifi.content.repository.class.name=DatabaseContentRepository
 ## define protocol parameters
 ## The default is CoAP, if that extension is built. 
 ## Alternatively, you may use RESTSender if http-curl is built
+#nifi.c2.agent.protocol.class=RESTSender
```

Comment:
This doesn't seem to be related

Reply:
it became obsolete in light of #1058, removed